wasm/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[target.wasm32-unknown-unknown]
```

Comment:
This is necessary, see https://docs.rs/getrandom/latest/getrandom/#webassembly-support